cns/restserver/ipam.go

```diff
@@ -86,7 +88,7 @@ func (service *HTTPRestService) requestIPConfigHandler(w http.ResponseWriter, r
 		}
 	}
 
-	reserveResp := &cns.IPConfigResponse{
+	reserveResp := cns.IPConfigResponse{
```

Comment:
I remember you were experimenting with this change to see if that helps with some error you were having. Is this change needed? I wouldn't think so